### PR TITLE
adding a graph component, adding styles for the homepage

### PR DESCRIPTION
**Pull request by yuliwna1**

### CODE REVIEW

File: src/components/Graphs/LineGraph.js [hunk 1]
```diff
@@ -1,6 +1,58 @@
...
+    name: 'cohort-8',
+    count: 43
+  },
+  {
+    name: 'cohort-7',
+    count: 44
+  }
+];
+
+export default class LineGraph extends PureComponent {
```
Comment (vindhya) [inline src/components/Graphs/LineGraph.js]: Does this have to be a PureComponent because of Rechart?

File: src/GlobalStyle.js [hunk 2]
```diff
@@ -14,7 +15,168 @@ const GlobalStyle = createGlobalStyle`
...
+
+export const Badge = styled.div`
+  border-radius: 5px;
+  background: rgb(235, 44, 151);
+  padding: 10px 20px;
+  ${fontColor};
+  ${fontSize};
+`;
+
+// export default GlobalStyle;
```
Comment (martapa) [inline src/GlobalStyle.js]: unnecessary comment

File: src/components/Graphs/LineGraph.js (hunk 1 above)
Comment (IrinaGM) [reply src/components/Graphs/LineGraph.js]: Found this: https://codeburst.io/when-to-use-component-or-purecomponent-a60cfad01a81 

It's interesting, I think that using the pure component here does add value; I think the graph will not be re-rendered each time that something changes in state or props from HomePage. hope I understood it correctly.

File: src/views/HomePage.js [hunk 3]
```diff
@@ -1,23 +1,26 @@
...
 export const HomePage = () => {
   return (
     <HomePageStyle>
-      <Title />
-      <CohortList cohorts={cohorts} />
-      <LineGraph />
+      <Wrapper column flex>
+        <Title />
+        <LineGraph />
+        <CohortList cohorts={cohorts} />
```
Comment (vindhya) [inline src/views/HomePage.js]: Should the CohortList component still have a scroll bar now that it's under the graph?

**Pull request merged**